sdk/core/Azure.Core/tests/samples/LoggingSamples.cs

```diff
@@ -35,7 +35,7 @@ public void LoggingCallback()
         {
             #region Snippet:LoggingCallback
             using AzureEventSourceListener listener = new AzureEventSourceListener(
-                (e, message) => Console.WriteLine($"{DateTime.Now} {message}"),
+                (e, message) => Console.WriteLine("[{0:HH:mm:ss:fff}] {1}", e.TimeStamp.ToLocalTime(), message),
```

Comment:
Note: turns out `EventWrittenEventArgs.TimeStamp` was added in netstandard2.1, netcoreapp2.2. Is this a concern for a sample? I could use something else that existed in netstandard2.0. The main goal was to show an example using what `e` contained, but also how to log the event timestamp (a very common thing for useful logging to help diagnose issues).

Reply:
Yep, it fails to build in CI.

Reply:
I could effectively change it back to use the current machine time, use something like `EventWrittenEventArgs.Level` like your default does, and make note that when targeting netstandard2.1 or netcoreapp2.2 (spelled out) you can use the `TimeStamp` to log the time the event was written instead of rendered. Or we target this project to netcoreapp2.2, but I'm doubting it's worth it.